core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraAdminRepairIntegrationTest.java

```diff
@@ -28,4 +28,8 @@ protected void initialize(String testName) {
     admin = new CassandraAdmin(clusterManager, new DatabaseConfig(properties));
     adminTestUtils = new CassandraAdminTestUtils(properties, clusterManager);
   }
+
+  protected boolean isTimestampTypeSupported() {
```

Comment:
`@Override`  is missing?

Reply:
Indeed, thank you. 
I revised it in https://github.com/scalar-labs/scalardb/pull/2301/commits/2ec00e4f705e12c30b9b52f8f830e6b06ba45314